.gitignore

```diff
@@ -36,6 +36,7 @@ pip-log.txt
 pip-delete-this-directory.txt
 
 # Unit test / coverage reports
+circleci_api_py.html
```

Comment:
shouldn't this just be a part of htmlcov?

Reply:
for me it gets created in the package root folder. I just moved it a few lines below if that's what you mean